### PR TITLE
fix(components): [el-form] vertical aligin

### LINKED ISSUE #4687: [bug report] El radio button used in radio group is not aligned in form inline mode
### Element Plus version
1.2.0-beta.5

### OS/Browsers version
Chrome

### Vue version
3.2.23

### Reproduction Link
https://codesandbox.io/s/element-plus-radio-group-form-inline-display-7ss4y?file=/src/App.vue

### Steps to reproduce
form设置inline， 
加入el-radio-group，
el-radio-group中使用el-radio-button
不能对齐

### What is Expected?
el-radio-button能对齐

### What is actually happening?
el-radio-button没对齐

### PR DESCRIPTION
**Pull request by Alanscut**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.
fix #4687

### CODE REVIEW

**Comment by YunYouJun** [comment]
`line-height` to `height` seems make

image

be

image

Can you take a screenshot of why you did this?

**Comment by Alanscut** [comment]
@YunYouJun No, I should not modify the line-height. I modified the submitted code, please review it.

**Review by sxzz — APPROVED** [review]

File: packages/theme-chalk/src/radio.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
   cursor: pointer;
   display: inline-flex;
   align-items: center;
+  vertical-align: middle;
```
Comment (YunYouJun) [inline packages/theme-chalk/src/radio.scss:20]: Do you add `vertical-align` to decrease the height of radio group?

**Review by YunYouJun — APPROVED** [review]

File: packages/theme-chalk/src/radio.scss (hunk 1 above)
Comment (YunYouJun) [reply packages/theme-chalk/src/radio.scss:20]: The radio group is `inline-block`, and the radio is `inline-flex`, should we use `inline-flex` for radio group?

If we do not need, just merge it.

Comment (Alanscut) [reply packages/theme-chalk/src/radio.scss:20]: Yes. I think both are fine.

**Comment by Alanscut** [comment]
@YunYouJun please review it.

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**